File: SWexpert/D2/1284/1284.java
import java.io.BufferedReader;
import java.io.IOException;
import java.io.InputStreamReader;

public class Solution {
	public static void main(String[] args) throws IOException {
		BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
		int T = Integer.valueOf(br.readLine());
		for (int i = 0; i < T; i++) {
			String st[] = br.readLine().split(" ");
            int p = Integer.valueOf(st[0]);
            int q = Integer.valueOf(st[1]);
            int r = Integer.valueOf(st[2]);
            int s = Integer.valueOf(st[3]);
            int w = Integer.valueOf(st[4]);
            int aCharge, bCharge;
            aCharge = p * w;
            if(w < r) bCharge = q;
            else bCharge = q + (w - r) * s;
            System.out.printf("#%d %d\n", i+1, (aCharge > bCharge) ? bCharge : aCharge);
		}
	}
}

// 스캐너에서 오류 있어서 줄단위로 받아와서 split으로 쪼개서 줄단위 핸들링

File: SWexpert/D2/1284/1284.cpp
#include <iostream>
using namespace std;

int main(){
    int T;
    cin >> T;
    
    // 삼항연산자는 if같은 논리식이 들어갈 수 없다 ((W * P) > (if(W <= R){return Q} else{return (W-R)*S +Q})) ? (W * P) : (if(W <= R){return Q} else{return (W-R)*S +Q}))
    
    for(int i = 0; i < T; i++){        
        int P, Q, R, S, W, sd;
    	cin >> P;
    	cin >> Q;
    	cin >> R;
    	cin >> S;
    	cin >> W;
        int aCharge, bCharge;
    	aCharge = P * W;
    	if(W < R){
	        bCharge = Q;
	    }
	    else
	    {
	        bCharge = Q + (W-R) * S;
	    }
	    (aCharge > bCharge) ? sd = bCharge : sd = aCharge;
        cout << "#" << i+1 << " " << sd << endl;
    }
    return 0;
}
